python/semantic_kernel/connectors/ai/bedrock/services/model_provider/bedrock_mistralai.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from typing import Any
+
+from semantic_kernel.connectors.ai.bedrock.bedrock_prompt_execution_settings import (
+    BedrockChatPromptExecutionSettings,
+    BedrockTextPromptExecutionSettings,
+)
+from semantic_kernel.connectors.ai.bedrock.services.model_provider.utils import remove_none_recursively
+from semantic_kernel.contents.text_content import TextContent
+
+# region Text Completion
+
+
+def get_text_completion_request_body(prompt: str, settings: BedrockTextPromptExecutionSettings) -> Any:
+    """Get the request body for text completion for Mistral AI models.
+
+    https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-mistral-text-completion.html
+    """
+    return remove_none_recursively({
+        "prompt": f"<s>[INST] {prompt} [/INST]",
+        "max_tokens": settings.max_tokens,
+        "stop": settings.stop,
+        "temperature": settings.temperature,
+        "top_p": settings.top_p,
+        "top_k": settings.top_k,
+    })
+
+
+def parse_text_completion_response(response: dict[str, Any], model_id: str) -> list[TextContent]:
+    """Parse the response from text completion for AI21 Labs models."""
+    return [
+        TextContent(
+            ai_model_id=model_id,
+            text=output["text"],
+            inner_content=output,
+        )
+        for output in response.get("outputs", [])
+    ]
+
+
+# endregion
+
+
+# region Chat Completion
+
+
+def get_chat_completion_additional_model_request_fields(
```

Comment:
In the underlying connector code, I see us being able to add multiple system messages; however for Mistral, they only allow 1 system message (in the specific Mistral connector we're taking the first system message if multiple are defined). How does that work here?

Reply:
I checked the Bedrock documentation and the MistralAI docs. I could not find mentions of single system message. I also tried a Mistral AI model with the Bedrock connector and two system message, it was able to return a response normally.

Reply:
I'm getting it mixed up with Anthropic... d'oh. Too many connectors. :) Disregard.